.gitignore

```diff
@@ -40,3 +40,8 @@ ExportedObj/
 
 # Unity3D Generated File On Crash Reports
 sysinfo.txt
+
+# test token file for dev purposes
+src/ApprienUnitySDK/Assets/dont_push_token.asset
```

Comment:
If only the token is not pushed to git, the game object will have a missing reference to the file. I think it's better to explicitly have None instead of Missing Reference in the scene files. Thus, when saving the example scene file, leave the field as None and don't commit a version that has a reference to a git-ignored file.